apps/src/Sound.js

```diff
@@ -315,33 +304,21 @@ Sound.prototype.fadeToGainHtml5Audio_ = function(gain, durationSeconds) {
 };
 
 Sound.prototype.getPlayableFile = function() {
-  // IE9 Running on Windows Server SKU can throw an exception on window.Audio
-  try {
-    if (!window.Audio) {
-      return false;
-    }
+  if (!window.Audio) {
+    return false;
+  }
 
-    var audioTest = new window.Audio();
+  var audioTest = new window.Audio();
```

Comment:
Probably a bigger discussion to figure it out, so maybe we can chat at check-in today - but, as tidy code in these areas, do we want to update to ES6 when we touch this code? In this case, I'm looking at updating this `var` to `let`.

Reply:
mmm good q -- looks like we're already mixing ES6 vs. not in this file, so I think it makes sense to update this since I touched it? Still a bunch more `var` declarations to update elsewhere, but I think can leave those for another time?

Reply:
That seems like a reasonable compromise that still moves us towards a cleaner codebase!
![otters_cleanup](https://user-images.githubusercontent.com/2959170/187984661-7f0ae395-dcea-4ae4-8746-a4c4ca24a024.gif)

Reply:
Just updated in [this commit](https://github.com/code-dot-org/code-dot-org/pull/47880/commits/d6dfd42fedf0270a3459c2d3b3c341eca542867f) -- quick check that I'm not crazy and can declare this as a `const`? Similar declaration used `let` in another function that I updated, so want to make sure I'm not missing anything. 

Tangentially, I want to share the docs for this function -- really spoke to my experience working with audio ("will this work? maybe?"):

![image](https://user-images.githubusercontent.com/25372625/188030995-90c74fe8-a548-41c3-b19c-0c12a4caca60.png)

Reply:
Looks like it can totally be a `const`! I have a bad habit of defaulting to `let`, so thank you for catching that!

And oh my gosh, that API is amazing. "Probably".